docs/DesignDocs/CustomAttestation.md

```diff
@@ -734,15 +734,40 @@ oe_result_t oe_verify_evidence(
     size_t* claims_length);
 
 /**
- * oe_free_claims_list
+ * oe_get_registered_attester_format_ids
  *
- * Frees a claims list.
+ * Get the unique identifiers of all registered attesters.
  *
- * @param[in] claims The list of claims.
- * @param[in] claims_length The length of the claims list.
+ * @param[out] format_ids The list of the UUIDs of the registered attesters.
+ * @param[out] format_ids_length The length of the UUIDs list.
  * @retval OE_OK on success.
  */
-oe_result_t oe_free_claims_list(oe_claim_t* claims, size_t claims_length);
```

Comment:
Why does this PR add and remove function definitions from documentation without actually modifying any code? For example, you're removing the `oe_free_claims_list` function from the doc but the function still exists.

Reply:
Accidentally removed oe_free_claims_list. Restored.

---
In reply to: [362996400](https://github.com/openenclave/openenclave/pull/2402#discussion_r362996400) [](ancestors = 362996400)